ports/hammerwatch/port.json

```
@@ -0,0 +1,32 @@
+{
+  "version": 4,
+  "name": "hammerwatch.zip",
+  "items": [
+    "Hammerwatch.sh",
+    "hammerwatch"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Hammerwatch",
+    "porter": [
+      "BinaryCounter"
+    ],
+    "desc": "Hammerwatch is a pixel-art hack-and-slash dungeon crawler where you battle through hordes of enemies, solve puzzles, and upgrade your character. It’s got a classic retro vibe, with different character classes, co-op gameplay, and tons of secrets to uncover. Think Gauntlet-style action with modern polish!",
+    "desc_md": null,
+    "inst": "This port is compatible with the 1.3 Steam release of the game.\nInstalling:\n - Open the Steam console and type in the following command to download the game files: \ndownload_depot 239070 239071 6038447391393479561\n - Put the gamefiles (without any surrounding folders) into hammerwatch/gamefiles/",
+    "inst_md": null,
+    "genres": [
+      "adventure",	
+      "action"
+    ],
+    "image": null,
+    "rtr": false,
+    "exp": false,
+    "runtime": ["weston_pkg_0.2","mono-6.12.0.122"],
```

Can't yet merge multi-runtime ports yet, but it should have the suffix of `.squashfs`. :D
```suggestion
    "runtime": ["weston_pkg_0.2.squashfs","mono-6.12.0.122.squashfs"],
```

---

Thanks. I didn't specify the suffix on the Super Hexagon port either and it seems to work and the port checker didn't flag this either. Should i update this on Hexagon too?